e2e/assertions/compareScreenshot.js

```diff
@@ -15,6 +17,26 @@ const ensureBaselinePhotoExists = (baselinePath, resultPath) => {
   }
 }
 
+const update = (baselinePath, resultPath, callback, data) => {
+  const read = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  })
+
+  read.question('Would you like to update the baseline for this screenshot? (y/n) ', answer => {
+    if (answer.toLowerCase() === 'y' || answer.toLowerCase() === 'yes') {
+      console.log('Generating new baseline screenshot...')
+      spawnSync('rm', [baselinePath], {
+        stdio: 'inherit',
+      })
+      fs.writeFileSync(baselinePath, fs.readFileSync(resultPath))
```

Comment:
Cool! You may not need to manually to the "rm" first though. 

According to the node docs for `writeFile`... https://nodejs.org/api/fs.html#fs_fs_writefile_file_data_options_callback
> Asynchronously writes data to a file, replacing the file if it already exists.

Reply:
Oh! Awesome! :)